client/my-sites/migrate/step-import-or-migrate.jsx

```diff
@@ -83,7 +83,10 @@ class StepImportOrMigrate extends Component {
 			return (
 				<p>
 					{ translate(
-						'You need to have {{jetpackInstallLink}}Jetpack{{/jetpackInstallLink}} installed on your site to be able to import everything.',
+						'You need to have Jetpack installed on your site to' +
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.